rescan.go

```diff
@@ -1085,8 +1109,15 @@ func DisableDisconnectedNtfns(disabled bool) UpdateOption {
 
 // Update sends an update to a long-running rescan/notification goroutine.
 func (r *Rescan) Update(options ...UpdateOption) error {
-	running := atomic.LoadUint32(&r.running)
-	if running != 1 {
+
+	uo := defaultUpdateOptions()
+	for _, option := range options {
+		option(uo)
+	}
+
+	select {
+	case r.updateChan <- uo:
+	case <-r.running:
```

Comment:
also select on `ro.quit` in case caller specified?

Reply:
Yep :)